packages/x-components/vue.config.js

```diff
@@ -2,6 +2,18 @@ const glob = require('glob');
 
 module.exports = {
   configureWebpack: {
-    entry: [...glob.sync('src/design-system/**/*.scss').map(style => `./${style}`), './src/main.ts']
+    entry: [
+      ...glob.sync('src/design-system/**/*.scss').map(style => `./${style}`),
+      './src/main.ts'
+    ],
+    output: {
+      devtoolModuleFilenameTemplate: ({ resourcePath, hash, query }) =>
```

Comment:
Since this is not likely to change very often here. I think it is just better to have the configuration hardcoded here.